tools/errortracker/errortracker.go

```diff
@@ -125,11 +125,19 @@ func handle(w http.ResponseWriter, r *http.Request) {
 		errorType += "-origin"
 	}
 	is3p := false
-	if r.URL.Query().Get("3p") == "1" {
-		is3p = true
-		errorType += "-3p"
+	runtime := r.URL.Query().Get("rt")
+	if runtime != "" {
+		errorType += "-" + runtime;
```

Comment:
this does not lengthen the string. we will have:

`(default|assert)-cdn-(1p|3p|inabox)[-canary]`